app/containers/SupportedVersions/useSupportedVersionMessage.ts

```diff
@@ -0,0 +1,46 @@
+import moment from 'moment';
+
+import { useAppSelector } from '../../lib/hooks';
+
+const applyParams = (message: string, params: Record<string, unknown>) => {
```

Comment:
I think it's worth a test or even a description of how this function works.

Reply:
Yeah, tested this hook here https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5185/files#diff-ef24f5e6b36852d882d4b1c3c8e5e36add01749f3a3bb743b8ce96d2c1c60cf7R70

It's very weird to have the hook tested on `SupportedVersionsWarning`, but it's per their documentation: https://github.com/testing-library/react-hooks-testing-library#when-not-to-use-this-library
> Your hook is defined alongside a component and is only used there
> Your hook is easy to test by just testing the components using it

Still, I started writing tests ignoring docs for the sake of readability of `hook.test.ts`, only to realize it's a *lot* easier to test using the RNTL rather than RHTL.
It makes too much use of Redux, so with a mounted component, it's just a matter of having a Provider and dispatching changes.
With the hooks lib, I'd have to recreate the whole state for every test.